### PR TITLE
Use rosa flag as a single line and parse mutliple value

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

File: app/addon_cmds/commands.py [hunk 1]
```diff
@@ -139,12 +139,13 @@ def addon(
     """
     Command line to Install/Uninstall Addons on OCM managed cluster.
     """
+    _rosa = [addon_name.strip() for addon_name in rosa.split(",")] if rosa else []
```
Comment (myakove) [inline app/addon_cmds/commands.py:142]: this is not needed, just do `_rosa = rosa.split(",")`

File: app/addon_cmds/commands.py [hunk 2]
```diff
@@ -168,12 +169,12 @@ def addon(
...
 
     ctx.obj["addons_dict"] = addons_dict
-    if any(addon_name not in addons_dict.keys() for addon_name in rosa):
+    if any(addon_name not in addons_dict.keys() for addon_name in _rosa):
         click.echo(
             f"""
 An addon indicated with --rosa does not match any of addons names that were given.
 Addons to install/uninstall: {', '.join(addons_dict.keys())}.
-Addons to use with rosa: {', '.join(rosa)}.
+Addons to use with rosa: {', '.join(_rosa)}.
```
Comment (myakove) [inline app/addon_cmds/commands.py:177]: No need to join, use rosa (it's str)

File: app/addon_cmds/commands.py [hunk 3]
```diff
@@ -168,12 +169,12 @@ def addon(
         )
 
     ctx.obj["addons_dict"] = addons_dict
-    if any(addon_name not in addons_dict.keys() for addon_name in rosa):
+    if any(addon_name not in addons_dict.keys() for addon_name in _rosa):
```
Comment (myakove) [inline app/addon_cmds/commands.py:172]: `addons_dict` keys is not needed.

File: app/addon_cmds/commands.py (hunk 1 above)
Comment (oharan2) [reply app/addon_cmds/commands.py]: Done, but note that it's space sensitive, not intuitive for all users.
example: 
`--rosa "managed-odh, managed-api-service"` **will raise and abort.**
`--rosa managed-odh,managed-api-service` will pass with/without `""`
@myakove @rnetser Please resolve if it's OK.

File: README.md [hunk 4]
```diff
@@ -93,17 +93,20 @@ podman run quay.io/redhat_msi/ocp-addons-operators-cli \
...
+    -a ocm-addon-test-operator-1 \
     -a ocm-addon-test-operator-2 \
+    -a ocm-addon-test-operator-3 \
     -c cluster-name \
-    --rosa ocm-addon-test-operator-2 \
+    --rosa ocm-addon-test-operator-1,ocm-addon-test-operator-3 \
     install
 ```
-Only addon `ocm-addon-test-operator-2` will be installed with ROSA cli.
+Only addons `ocm-addon-test-operator-1`, `ocm-addon-test-operator-3` will be installed with ROSA cli.
```
Comment (myakove) [inline README.md:109]: Only addons `ocm-addon-test-operator-1` and `ocm-addon-test-operator-3`

**Review by rnetser — APPROVED** [review]

**Pull request merged**